packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Customization.Example.tsx

```diff
@@ -1,219 +1,218 @@
 import * as React from 'react';
-import { DirectionalHint, ContextualMenuItemType } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { ContextualMenuItemType, DirectionalHint, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
 import { DefaultButton, IconButton } from 'office-ui-fabric-react/lib/Button';
 import { FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
-import './ContextualMenuExample.scss';
 
-export class ContextualMenuCustomizationExample extends React.Component<{}, {}> {
-  public render(): JSX.Element {
-    return (
-      <DefaultButton
-        className="ContextualMenuButton3"
-        text="Click for ContextualMenu"
-        menuProps={{
-          shouldFocusOnMount: true,
-          directionalHint: DirectionalHint.bottomLeftEdge,
-          className: 'ms-ContextualMenu-customizationExample',
-          items: [
+export const ContextualMenuCustomizationExample: React.FunctionComponent = () => {
+  return (
+    <DefaultButton
+      className="ContextualMenuButton3"
+      text="Click for ContextualMenu"
+      menuProps={{
+        shouldFocusOnMount: true,
+        directionalHint: DirectionalHint.bottomLeftEdge,
+        className: 'ms-ContextualMenu-customizationExample',
+        items: menuItems
+      }}
+    />
+  );
+};
+
+function renderCharmMenuItem(item: any, dismissMenu: () => void): JSX.Element {
+  return (
+    <IconButton
+      iconProps={{ iconName: item.name }}
+      className="ms-ContextualMenu-customizationExample-icon ms-ContextualMenu-link"
+      data-is-focusable={true}
+      onClick={dismissMenu}
+    />
+  );
+}
+
+function renderCategoriesList(item: any): JSX.Element {
```

Comment:
What's the type of `item`(here and elsewhere in the file, and a couple others)?

Reply:
It is really `any` in the types file but doesn't matter if we change it to be `IContextualMenuItem` here. I'm also enforcing what category list is by creating an interface in the file.

Reply:
Oh okay. In that case I'm fine with either typed or any.